### ISSUE TITLE
[1.10.2] Question about Loot Table

### ISSUE THREAD

**Issue by ARatfat** [Ipsis/Woot#136]
Hello, 

First Thanks for your mod it's trully server friendly. 

I use headcrumbs & In Control mod (add curstom mod settup). The prism can detect the custum mob from headcrumbs but I cant insert them to controleur. maybe I can "teach" to woot the "new". 
If I creat first custom mobs if loot.json can be configured auto with new rules of In control.

**Comment by Ipsis** [comment]
I'm not sure Woot and the loot.json is going to do what you need it to do. But I will know better when I see how Headcrumbs and In Control actually work.

I also need to see why the prism is rejected by the controller.

**Comment by ARatfat** [comment]
In fact when I capture a headcromb mob with a pism. Prism detect it with syntaxe Woot:none:headcrumbs.Human 
More the prism detect the mob but I can't take the prism in controleur

**Comment by Ipsis** [comment]
I've installed Headcrumbs into my test area and spawned a human.
It became Elooram and I captured that as Woot:none:headcrumbs.Human
I've now installed it into a tier I factory and it is generating loot.
eg. Music disks, snowballs, rotten flesh, redstone torch etc.

No chance that someone has blacklisted Headcrumb mobs on you instance?

However I think you are going to run into an issue if you want to get specific loot from different Headcrumb mobs. My guess is that ALL Headcrumb mobs are going to by Woot:none:headcrumbs.Human regardless of what person they are for. Woot has no other information other than that "Woot:none:X" name to identify different mobs.
